packages/client/hmi-client/src/api/api.ts

```diff
@@ -80,6 +80,7 @@ export interface PollResponse<T> {
 	error: any;
 	progress?: any;
 	data: T | null;
+	cancelled?: boolean;
```

Comment:
This is how I am enabling the simulation-service.ts `pollAction` to fwd the cancel state to this as previously it was only able to be done with the object itself using `stop()`